### PR TITLE
MAISTRA-1720 Fix CRD generation for SMCPv2

### PR DESCRIPTION
**Pull request by dgn**
Apparently Kubernetes 1.18 introduces CRD validation that fails on core types :face_with_head_bandage: 

This is the quick fix to get things working on OCP 4.5+, the real fix would be to change either the core type or the CRD validation code in K8s

Note that I didn't re-generate the CRDs in this PR because it hasn't been done for a while and would introduce a lot of other changes. I did test this though

### CODE REVIEW

**Comment by dgn** [comment]
`gen-check` fails until https://github.com/maistra/istio-operator/pull/504 is merged

**Review by rcernich — APPROVED** [review]

File: build/generate-manifests.sh [hunk 1]
```diff
@@ -45,6 +45,10 @@ function checkDependencies() {
 
 function generateCRDs() {
   go run -mod=vendor github.com/operator-framework/operator-sdk/cmd/operator-sdk generate crds --crd-version v1
+
+  # workaround for https://github.com/kubernetes-sigs/controller-tools/issues/457
```
Comment (jwendell) [inline build/generate-manifests.sh:49]: Should we create an issue to track this?

**Review by jwendell — APPROVED** [review]

File: build/generate-manifests.sh (hunk 1 above)
Comment (dgn) [reply build/generate-manifests.sh:49]: Good point. I filed https://issues.redhat.com/browse/MAISTRA-1721

**Pull request merged**